### ISSUE TITLE
Flake: oc set trigger

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin#14180]
https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_origin/639/testReport/junit/github/com_openshift_origin_test_cmd_triggers_buildconfigs/test_cmd_triggers_sh_48__executing__oc_set_triggers_bc_ruby_hello_world___from_webhook__expecting_success_and_text__updated_/

**Comment by csrwng** [comment]
@juanvallejo this is the issue I mentioned. You seem to have a fix in hand.

**Comment by juanvallejo** [comment]
@csrwng Thanks! Currently waiting on [upstream PR](https://github.com/kubernetes/kubernetes/pull/45918), but it does address this flake

**Comment by mfojtik** [comment]
seen here: https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/2753/testReport/junit/github/com_openshift_origin_test_cmd_triggers_buildconfigs/test_cmd_triggers_sh_58__executing__oc_set_triggers_bc_ruby_hello_world___from_gitlab__expecting_success/

**Comment by juanvallejo** [comment]
Origin PR that addresses this: https://github.com/openshift/origin/pull/14211

@liggit if we want to wait to merge the rest of the PR, I could pick the changes that just address this issue into a new PR

**Comment by juanvallejo** [comment]
I believe the patch for [set/triggers.go](https://github.com/openshift/origin/pull/14211/commits/332d67fb5c31c0d92bbcc2acb6dfa73325c217c0#diff-bd70f138bff1cf8a88aaa6397942eec6L229) (which was causing this flake) has already [made it into origin](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/set/triggers.go#L227). (The `SelectorParam` and `ResourceTypeOrNameArgs` were being called a second time under the `!o.Local` condition). Will go ahead and rebase #14211 with latest master.

Closing this issue as I believe it has been resolved. Please re-open if flake continues to happen.
